frontend/src/metabase/parameters/utils/dashboards.ts

```diff
@@ -66,6 +66,26 @@ export function setParameterName(
   };
 }
 
+export function getSourceType(parameter: Parameter): ParameterSourceType {
+  return parameter.source_type ?? null;
+}
+
+export function getSourceOptions(parameter: Parameter): ParameterSourceOptions {
+  return parameter.source_options ?? {};
+}
```

Comment:
Should we just do these checks inline relying on TypeScript? I'm worried these helpers might get lost as they're not doing anything you can't do with regular JS operators